Test/fmi_testutil.h

```diff
@@ -21,6 +25,12 @@ void fmi_testutil_build_xml_path(char* buf, size_t bufSize, const char* basePath
  */
 void fmi_testutil_enter_breakpoint();
 
+/**
+ * Parses the modelDescription.xml which is located in 'xmldir'. Expects successful parsing.

```

Comment:
It's in a bit strange state right now. There are ASSERT_MSG macros which do not kill the test, but print and return 0 (or null - depending on contexts since it's a macro). We really need to clean that up. I removed the 'Expects successful parsing' part since it's not REQUIRED.